unix/wheels/build-wheels-docker.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2010-2016 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+if [ $GEM_SET_DEBUG ]; then
+    set -x
+fi
+set -e
+
+yum -y install autoconf curl gzip libtool tar unzip
+
+if [ $GEM_SET_NPROC ]; then
+    NPROC=$GEM_SET_NPROC
+else
+    #Everyone has at least two cores
+    NPROC=2
+fi
+OQ_PREFIX=/build
+
+rm -Rf $OQ_PREFIX
+mkdir -p $OQ_PREFIX/src
+
+cd $OQ_PREFIX/src
+
+# Get sources
+curl -Lo libgeos-3.5.0.tar.gz https://github.com/libgeos/libgeos/archive/3.5.0.tar.gz
+curl -LO http://www.hdfgroup.org/ftp/HDF5/current/src/hdf5-1.8.17.tar.gz
+curl -LO https://pypi.python.org/packages/22/82/64dada5382a60471f85f16eb7d01cc1a9620aea855cd665609adf6fdbb0d/h5py-2.6.0.tar.gz
+curl -LO https://pypi.python.org/packages/ad/f9/4640d50324635fbdc7b109f8ef37de5f04456b89ed175cf2f71ae05efd8f/Shapely-1.5.13.tar.gz
```

Comment:
Why? That looks quite over engineered. This scripts does not run every day, is only needed if we loose the whl files or if we need to build a new version and in that case we need to upgrade the script anyway.